client/landing/gutenboarding/onboarding-block/acquire-intent/arrow.tsx

```diff
@@ -3,8 +3,15 @@
  */
 import React from 'react';
 
-const Arrow: React.FunctionComponent = () => (
-	<svg width="26" height="18" viewBox="0 0 26 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+const Arrow: React.FunctionComponent< React.SVGProps< SVGSVGElement > > = props => (
+	<svg
+		{ ...props }
+		width="26"
+		height="18"
+		viewBox="0 0 26 18"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+	>
 		<path d="M16 1L24 9M24 9L16 17M24 9H0" stroke="currentColor" strokeWidth="2" />
 	</svg>
```

Comment:
This should use `@wordpress/components` `SVG` and `Path`. Ok to leave that out of scope fo this PR.

Reply:
Thanks! TIL.